pkg/app/app.go

```diff
@@ -37,6 +38,9 @@ func NewCR(c Config) *applicationv1alpha1.App {
 		if !c.DisableForceUpgrade {
 			annotations["chart-operator.giantswarm.io/force-helm-upgrade"] = "true"
 		}
+		if c.PauseReconciliation {
+			annotations["app-operator.giantswarm.io/paused"] = "true"
```

Comment:
Can we use the constant Pawel is adding? https://github.com/giantswarm/apiextensions/pull/649

Reply:
Sure thing.